static/app/components/globalEventProcessingAlert.tsx

```diff
@@ -0,0 +1,51 @@
+import {Fragment} from 'react';
+
+import Alert from 'app/components/alert';
+import ExternalLink from 'app/components/links/externalLink';
+import {IconInfo} from 'app/icons';
+import {tct} from 'app/locale';
+import {Project} from 'app/types';
+
+const sentryStatusPageLink = 'https://status.sentry.io/';
+
+type Props = {
+  projects: Project[];
+  className?: string;
+};
+
+// This alert makes the user aware that one or more projects have been selected for the Low Priority Queue
+function GlobalEventProcessingAlert({className, projects}: Props) {
+  const projectsInTheLowPriorityQueue = projects.filter(
+    project => project.eventProcessing.symbolicationDegraded
+  );
+
+  if (!projectsInTheLowPriorityQueue.length) {
+    return null;
+  }
+
+  return (
+    <Alert className={className} type="info" icon={<IconInfo size="sm" />}>
+      {projectsInTheLowPriorityQueue.length === 1
+        ? tct(
+            'Event Processing for this project is currently degraded. Events may appear with larger delays than usual or get dropped. Please check the [link:Status] page for a potential outage.',
+            {
+              link: <ExternalLink href={sentryStatusPageLink} />,
+            }
+          )
+        : tct(
+            'Event Processing for the [project-slugs] projects is currently degraded. Events may appear with larger delays than usual or get dropped. Please check the [link:Status] page for a potential outage.',
+            {
+              'project-slugs': projectsInTheLowPriorityQueue.map(({slug}, index) => (
```

Comment:
I think we usually use camelCase for these
```suggestion
              'projectSlugs': projectsInTheLowPriorityQueue.map(({slug}, index) => (
```